runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -293,70 +293,208 @@ initImpl(J9VMThread *currentThread, j9object_t membernameObject, j9object_t refO
 	}
 }
 
-static char *
+typedef struct LocalJ9UTF8Buffer {
+	/**
+	 * Constructs an empty LocalJ9UTF8Buffer.
+	 */
+	LocalJ9UTF8Buffer() = default;
```

Comment:
This doesn't do as advertised: it just creates an uninitialized object. Instead it should initialize all fields: `capacity` should be zero and the pointers should be null.